modules/Xsens_module/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BIND
   USE_LINK_PATH)
 
 # Find required package
-find_package(ICUB REQUIRED)
+#find_package(ICUB REQUIRED)
```

Comment:
If iCub is not required you can remove the line

Reply:
done!